test/development/acceptance-app/unsupported-app-features.test.ts

```diff
@@ -30,7 +36,10 @@ createNextDescribe(
       `
       )
 
-      expect(await session.hasRedbox(true)).toBe(true)
+      await check(
+        async () => ((await session.hasRedbox(true)) ? 'redbox' : 'nothing'),
```

Comment:
I checked the source code of `hasRedbox()` and it already waits for 30 seconds

(or rather, waits 1 second 30 times)

https://github.com/vercel/next.js/blob/9791d1e60853b408c0eb1d19a43ee01614797db3/test/lib/next-test-utils.js#L676

I think `check()` here is probably not necessary?

Reply:
Yeah check here would be redundant and just increasing the number of attempts.